examples/demo/gloo/offscreen.py

```diff
@@ -103,7 +103,7 @@ def __init__(self, size=(600, 600)):
         app.Canvas.__init__(self, show=False, size=size)
         self._t0 = time()
         # Texture where we render the scene.
-        self._rendertex = gloo.Texture2D(self.size)
+        self._rendertex = gloo.Texture2D(shape=self.size + (4,))
```

Comment:
So it turns out the error on intel platforms is that the renderbuffer we were creating wasn't complete -- I guess it was creating a luminance instead of RGB(A) texture here.

In any case, the GL spec says that it's strongly recommended to check that framebuffers are complete before rendering to them:

https://www.khronos.org/opengles/sdk/docs/man/xhtml/glCheckFramebufferStatus.xml

We currently don't have that implemented. I plan to put it in `gloo` to hopefully save us some future pain.